tests/lib/k8s/test_builders.py

```diff
@@ -176,17 +176,50 @@ def test_get_anti_affinity_zone_enabled_for_process(self, parse_yaml_mock):
 
     def test_get_image_name_branch(self):
         """Returns the URL of the docker image if tag and branch are provided"""
-        app_config = {"app": "app", "registry": {"organization": "my-organization"}}
+        app_config = {
+            "app": "app",
+            "registry": {"platform": "docker-hub", "id": "id-1"},
+            "docker": {
+                "registries": {
+                    "docker-hub": [
+                        {"id": "id-1", "organization": "my-organization-1"},
+                        {"id": "id-2", "organization": "my-organization-2"},
+                    ],
+                },
+            },
+        }
         image_url = k8s_builders.get_image_name(
             app_config, {"branch": "feature/api", "tag": "1.0.0-sha-a2b3c4"}
         )
-        self.assertEqual(image_url, "my-organization/app:feature/api")
+        self.assertEqual(image_url, "my-organization-1/app:feature/api")
```

Comment:
It does not make sense to have this return the first and not the second one IMO, that's why we should keep it simple